qpup_hw/CMakeLists.txt

```diff
@@ -5,8 +5,10 @@ find_package(catkin REQUIRED COMPONENTS
         hardware_interface
         joint_limits_interface
         roscpp
+        tf2
         transmission_interface
         )
+find_package(Threads REQUIRED)
```

Comment:
lol does it not compile without this?

Reply:
Nope, need -pthreads. In most ros stuff, people just get it transitively lol